README.md

```diff
@@ -355,7 +355,7 @@ module.exports = {
 };
 ```
 
-#### `Function`
+#### `function`
```

Comment:
Also @alexander-akait  there is no such type as function here but it would be better if we have this heading as function? Is this okay? (This is the heading for the part that shows the function example of the option)